src/libraries/Microsoft.Extensions.Options/gen/Emitter.cs

```diff
@@ -414,19 +485,20 @@ public void EmitRangeAttribute(string modifier, string prefix, string className,
                 string.Format(global::System.Globalization.CultureInfo.CurrentCulture, GetValidationErrorMessage(), name, Minimum, Maximum);
         private bool NeedToConvertMinMax { get; }
         private bool Initialized { get; set; }
+        private const string c_minMaxError = "The minimum and maximum values must be set to valid values.";
```

Comment:
I would recommend extracting all error messages to a standalone source file.

Reply:
We can do that separately to reduce the changes we are porting to servicing.